### DISCUSSION TITLE
Is there a way to get it to hide all Signal contacts?

### DISCUSSION THREAD

**Discussion by dneu** [isamert/scli#226] [Ideas]
I have a list of people I haven't spoken to in years -- or ever on Signal. Is there an option to hide them, or can there be one?

**Comment by exquo** [comment]
An option like `--hide-contacts=<contacts_IDs>` can be added.

A more general solution would be to allow users to create folders where contacts could be placed (e.g. "Archived" / "Old", "Family", etc). But that would require allowing for a directory structure browsing in the UI, with folding, etc.
